### PR TITLE
Name actor flags 1: `ACTOR_FLAG_REACT_TO_LENS`, `ACTOR_FLAG_IGNORE_POINT_LIGHTS`

### PR DESCRIPTION
**Pull request by Dragorn421**
These are the lowest hanging fruits in actor flags
I'd like to see the rest documented together with their respective relevant functions/code (hookshot, sfx, targeting...)

Reference for `ACTOR_FLAG_REACT_TO_LENS`:

- MM's flags https://github.com/zeldaret/mm/blob/faba61d9b8bd01c8cf8ff070cdb14f63fba253c8/include/z64actor.h#L487C4
- MM lens PR https://github.com/zeldaret/mm/pull/1123 ( https://github.com/zeldaret/mm/pull/1123#discussion_r996256978 )
- OoT limbo actor flags PR https://github.com/zeldaret/oot/pull/1122 ( https://github.com/zeldaret/oot/pull/1122#discussion_r858936878 )

Reference for `ACTOR_FLAG_IGNORE_POINT_LIGHTS`:
- OoT limbo actor flags PR https://github.com/zeldaret/oot/pull/1122

### CODE REVIEW

File: include/z64actor.h
Comment (fig02) [inline include/z64actor.h:156]: I personally prefer to keep the comments on the side when short enough (which most of them will be small enough to fit, as seen in my old flags pr)
for example 
```c
#define ACTOR_FLAG_REACT_TO_LENS (1 << 7) // hidden or revealed by Lens of Truth (depending on room lensMode)
```
If not, and we want to keep all on top, I need line breaks between flags for my eyes to easily see which comment belongs to what
like so
```c
// hidden or revealed by Lens of Truth (depending on room lensMode)
#define ACTOR_FLAG_REACT_TO_LENS (1 << 7)

// comment
#define ACTOR_FLAG_8 (1 << 8)

// comment
#define ACTOR_FLAG_9 (1 << 9)

// comment
#define ACTOR_FLAG_10 (1 << 10)

// comment
#define ACTOR_FLAG_ENKUSA_CUT (1 << 11)

// Actor will not shake when a quake occurs
#define ACTOR_FLAG_IGNORE_QUAKE (1 << 12)
```

Comment (fig02) [reply include/z64actor.h:156]: (the only one thats really long and cant fit in a one liner so far is `ACTOR_FLAG_TALK` in another PR, and for that one I will also advocate for a line break either now or sometime after its merged)

Comment (Dragorn421) [reply include/z64actor.h:156]: I took this style from MM's z64actor.h for sync purposes, personally I'm fine doing it differently, either same line or with more line breaks

Another option since these comments are doc comments anyway could be

```c
/**
 * Comment
 */
#define blah1
/**
 * Comment
 */
#define blah2
```

Comment (fig02) [reply include/z64actor.h:156]: think even with block comments id want line breaks between each flag.
Im fine with the current suggestion with line breaks added, but block comments are fine too if people want

Comment (engineer124) [reply include/z64actor.h:156]: I think with the line breaks, everything else in this PR is good for me

Comment (fig02) [reply include/z64actor.h:156]: believe all we're waiting for on this is the line breaks between flags and its good to go

Comment (fig02) [reply include/z64actor.h]: went ahead and made this change

**Review by fig02 — APPROVED** [review]

**Comment by fig02** [comment]
Merging without a second approval because engineer said "I think with the line breaks, everything else in this PR is good for me"
And the line breaks are now in.

**Pull request merged**
